src/components/app/App.tsx

```diff
@@ -35,41 +40,8 @@ const theme = createMuiTheme({
   }
 });
 
-// Temporary ErrorBoundary to use before bugsnag API key is accessed via environment variables
-// TODO: delete this ErrorBoundary and replace with bugsnag ErrorBoundary
-class ErrorBoundary extends React.Component<{}, { hasError: boolean }> {
-  public static getDerivedStateFromError() {
-    // Update state so the next render will show the fallback UI.
-    return { hasError: true };
-  }
-
-  private constructor(props: any) {
-    super(props);
-    this.state = { hasError: false };
-  }
-
-  public render() {
-    if (this.state.hasError) {
-      // You can render any custom fallback UI
-      return <h1>Something went wrong.</h1>;
-    }
-
-    return this.props.children;
-  }
-}
-
 // These wrapper components are temporary components to pass their children the value of contexts using the useContext hook
-// TODO: refactor Admin, ConfigDrop, PatchContainer and BuildView to be functional components that consume their respective contexts
-const AdminWrapper = () => {
-  const { apiClient } = useContext(ApiClientContext);
-  return <Admin APIClient={apiClient} />;
-};
-const Config = () => {
-  const {
-    actions: { updateConfig }
-  } = useContext(ApiClientContext);
-  return <ConfigDrop updateClientConfig={updateConfig} onLoadFinished={null} />;
-};
```

Comment:
prod and staging variables can no longer be set in dev environment. I tried setting the api and ui url equal to the staging urls while in dev and i got a 404 and CORS error... if we care about being able to point dev to staging and prod urls then we can create a ticket to figure it out.